django_versioned_static_url/templatetags/versioned_static.py

```diff
@@ -20,25 +19,25 @@ def versioned_static(file_path):
     """
 
     full_path = find(file_path)
-    url = static(file_path)
 
+    if not full_path:
+        msg = 'Could not find static file: {0}'.format(file_path)
+        raise Exception(msg)
```

Comment:
Actually in this case I think it should mask this error, as normal broken links in the page wouldn't throw an error. We should display a warning in the log and simply return the URL path un-versioned.